lib/Doctrine/Migrations/AbstractMigration.php

```diff
@@ -60,11 +60,22 @@ public function isTransactional() : bool
         return true;
     }
 
+    /**
+     * Description of this migration.
+     *
+     * @return string
+     */
     public function getDescription() : string
     {
         return '';
     }
 
+    /**
+     * Warn with a message if some condition is met.
```

Comment:
That's made fairly clear just by reading the signature (you don't have to read the code inside the method)

Reply:
Fixed